### ISSUE TITLE
delegaters, validators, candidates

### ISSUE THREAD

**Issue by jolesbi** [cosmos/cosmos-ui#223]
something i have been thinking about lately is the relationship between delegaters, validators, and candidates OR how we should think about these types of users...

currently, delegates (candidates and validators) have their own list and UI. validators also have their own list and UI. it seems to me that we should combine these lists into one list called "Delegates".

1. what is the value of keeping a separate list of validators only without the ability to delegate to them in that screen?

2. if we use the word "Delegates" to describe Candidates and Validators, let's use the word "Bond" as the action / verb.

also open to other words that describe both candidates AND validators.

**Comment by nylira** [comment]
I agree with both points. Renaming and merging candidates/validators simplifies the app, and using "Delegates" as the noun form only makes the app more clear. I want to explain how we ended up with the current layout though:

I've been using delegate in the verb form, as a way of handing responsibility of your atoms to a validator. (that's why I changed Delegates to Delegate in #218, was wondering why you changed it to Delegates, now I know) E.g. "I delegate atoms to a candidate". And that's why PageDelegate.vue currently means "the page where I delegate atoms", not a detail view for a delegate.

There are two other words in the UI we've been using for "delegate", which is "stake" and "allocate". Stake is used only in once place, in the sidebar menu title. "Allocate" is being used on PageDelegate, as a pre-step before bonding, where you allocate your atoms per-candidate.
